client/src/main/java/io/confluent/kafka/schemaregistry/client/rest/entities/requests/RegisterSchemaResponse.java

```diff
@@ -63,11 +155,38 @@ public boolean equals(Object o) {
       return false;
     }
     RegisterSchemaResponse that = (RegisterSchemaResponse) o;
-    return id == that.id;
+    return Objects.equals(version, that.version)
+        && id == that.id
+        && Objects.equals(schemaType, that.schemaType)
+        && Objects.equals(references, that.references)
+        && Objects.equals(metadata, that.metadata)
+        && Objects.equals(ruleSet, that.ruleSet)
+        && Objects.equals(schema, that.schema);
   }
 
   @Override
   public int hashCode() {
-    return Objects.hash(id);
+    return Objects.hash(schemaType, references, metadata, ruleSet, version, id, schema);
+  }
+
+  @Override
+  public String toString() {
+    StringBuilder buf = new StringBuilder();
+    buf.append("{");
+    if (version != null) {
+      buf.append("version=").append(version).append(", ");
+    }
+    buf.append("id=").append(id).append(", ");
+    buf.append("schemaType=").append(this.schemaType).append(",");
```

Comment:
do we need a whitespace ' ' after ',' here? like line 179

Reply:
same question for line 181-185

Reply:
Fixed